### ISSUE TITLE
EPI datasets, not getting published results

### ISSUE THREAD

**Issue by leannmlindsey** [MAGICS-LAB/DNABERT_2#99]
Hello,

Thank you for posting the GUE+ datasets. Unfortunately, I am not getting the same results as your published results when I run DNABERT2 on them. I believe in the supplemental, you state that you reported MCC for the EPI dataset.  (also, I am not 100% sure of the mapping between your reported EPI datasets of 0-1 and the listed datasets in the GUE+ data (GM12878, HUVEC, HeLa-S3, IMR90, K562, NKEK)

Our results:

DNABERT2 on GM12878
acc: 0.6845, f1: 0.6806000467204687, mcc: 0.3622498911964387
you reported MCC: 61.91 for dataset 0
 
DNABERT2 on HeLa-S3
acc: 0.8995, f1: 0.8994577471182263, mcc: 0.7997480886818011
you reported MCC: 79.19 for dataset 1
 
DNABERT2 on K562
acc: 0.7145, f1: 0.7066583167889823, mcc: 0.43058726256329627
you reported MCC: 83.50 for dataset

Can you share the hyperparameters that you used for these results, or is it possible that they are accuracy results and not mcc? 

Thank you for your help.
LeAnn

**Comment by Zhihan1996** [comment]
The reported results are mcc. You can see the following figure for some of our validation results during training.

<img width="1165" alt="image" src="https://github.com/MAGICS-LAB/DNABERT_2/assets/43626056/b97964f8-5dcf-4568-9d97-d206c5c3cbf7">

For the hyperparameters, we used 2e-5 lr and 32 batch sizes.

**Comment by leannmlindsey** [comment]
Thank you for your response. I will try again with these hyperparameters

**Comment by leannmlindsey** [comment]
Can you clarify which datasets correspond with 0, 1, 2, 3, 4, 5?

**Comment by blaugi** [comment]
Hello @leannmlindsey, 
Did you reach a hyperparameter configuration to replicate the paper's results? If so, could you share them please.
 I'm running into the same issues aforementioned by you.
